decidim-initiatives/app/services/decidim/initiatives/initiative_search.rb

```diff
@@ -50,13 +49,15 @@ def search_state
         answered = state.member?("answered") ? query.answered : nil
         open = state.member?("open") ? query.open : nil
         closed = state.member?("closed") ? query.closed : nil
+        draft = state.member?("created") ? query.created : nil
 
         query
           .where(id: accepted)
           .or(query.where(id: rejected))
           .or(query.where(id: answered))
           .or(query.where(id: open))
           .or(query.where(id: closed))
+          .or(query.where(id: draft).where(published_at: nil))
```

Comment:
Do you think would be possible to do something like `query.where(id: [accepted, answered, open, closed, draft])`?

Reply:
I think that the last conditional is different from the others.
Maybe this query has a better approach making the query like that:
`
query.where(id: [accepted, rejected, answered, open, closed]).or(query.where(id: draft, published_at: nil))
`